src/views/Communities/Tabs.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+
+import {
+    DropdownMenu,
+    Grid,
+    Spinner,
+    Tab,
+    TabList,
+    TabPanel,
+    Tabs,
+} from '@impact-market/ui';
+
+import Community from './Community'
+import String from '../../libs/Prismic/components/String';
+
+
+const Tabens = ({setStatusFilter, setActiveTab, activeTab, statusFilter, communities, communitiesTabs, loading}: any) => {
+    
+    //  Make status CamelCase
+    const firstLetter = statusFilter.charAt(0).toUpperCase()
+    const restOfText = statusFilter.slice(1)
+    const status = firstLetter + restOfText
+    
+    const dropdownItems = [
+        {
+            icon: 'check',
+            onClick: () => setStatusFilter('valid'),
+            title: 'Valid'
+        },
+        {
+            icon: 'loader',
+            onClick: () => setStatusFilter('pending'),
+            title: 'Pending'
+        },
+        {
+            icon: 'trash',
+            onClick: () => setStatusFilter('removed'),
+            title: 'Removed'
+        }
+    ];
+
+    return (
+        <>
+        <Tabs>
+                <TabList>
+                    <Tab
+                        onClick={() => setActiveTab('all')}
+                        title={<String id="all"/>}
+                    />
+                    <Tab
+                        onClick={() => setActiveTab('myCommunities')}
+                        title={<String id="myCommunities"/>}
+                    />
+                </TabList>
+
+                {activeTab === 'myCommunities' &&
+                    <DropdownMenu
+                        asButton
+                        icon="chevronDown"
+                        items={dropdownItems}
+                        title={status}
+                        wrapperProps={{
+                            mt:1
+                        }}
+                    />
+                }
+
+                {!!Object.keys(communities).length &&
+                    communitiesTabs.map((key: number) => (
+                        <TabPanel key={key}>
+                            {loading ? (
```

Comment:
Shouldn't the "loading" check be made outside the map?

Reply:
in this case I still want to show the tabs and the loading effect inside the tab. this way while users are navigating through tabs they're not getting spammed with a full screen with a loading icon. it only shows inside the tab area (where the grid with the list of communities is)